aula-pratica-04/ex2.c

```diff
@@ -0,0 +1,25 @@
+/*
+    Declare uma matriz 5 x 5. Preencha com 1 a diagonal principal e 
+    com 0 os demais elementos. Escreva ao final a matriz obtida.
+*/
+
+#include<stdio.h>
+
+int main()
+{
+    int matriz[5][5];
+
+    for(int i = 0; i < 5; i++) {
+        for(int j = 0; j < 5; j++) {
+            (j == i) ? matriz[i][i] = 1 : matriz[i][j] = 0;
```

Comment:
Puts man, se vc ta usando o ternary operator pra fazer um if, faça um `if` mesmo, ficar usando o ternary de if é um caminho para deixar a base de código bem confusa ao longo do tempo.

ou a gente usa o ternary como ele foi feito pra ser usado:

```suggestion
            matriz[i][i] = (j == i) ? 1 : 0;
```

ou a gente usa um if:
```suggestion
            if (j == i) {
                matriz[i][i] = 1
            else {
                matriz[i][j] = 0;
            }
```